docs/algorithms/airl.rst

```diff
@@ -14,6 +14,59 @@ Notes
 -----
 - AIRL paper: `Learning Robust Rewards with Adversarial Inverse Reinforcement Learning <https://arxiv.org/abs/1710.11248>`_
 
+Example
+=======
+
+Detailed example notebook: `4_train_airl.ipynb <https://github.com/HumanCompatibleAI/imitation/blob/master/examples/4_train_airl.ipynb>`_
+
+.. testcode::
+
+    import gym
+    from stable_baselines3 import PPO
+    from stable_baselines3.common.evaluation import evaluate_policy
+    from stable_baselines3.common.vec_env import DummyVecEnv
+    from stable_baselines3.ppo import MlpPolicy
+
+    from imitation.algorithms.adversarial.airl import AIRL
+    from imitation.data import rollout
+    from imitation.data.wrappers import RolloutInfoWrapper
+    from imitation.rewards.reward_nets import BasicShapedRewardNet
+    from imitation.util.networks import RunningNorm
+
+    env = gym.make("seals/CartPole-v0")
+    expert = PPO(policy=MlpPolicy, env=env)
+    expert.learn(1000)
+
+    rollouts = rollout.rollout(
+        expert,
+        DummyVecEnv([lambda: RolloutInfoWrapper(gym.make("seals/CartPole-v0"))] * 5),
```

Comment:
I know this is just copied from the notebooks, but I think it'd be better to use `util.make_vec_env` instead here. Both I think `util.make_vec_env("seals/CartPole-v0", n_envs=5)` is simpler (and introduces a useful utility method), and *more correct* as it will seed each of the environments separately and deterministically.

I'd suggest making that change in both the examples and source notebook.